packages/application-shell/src/components/fetch-project/fetch-project.js

```diff
@@ -12,6 +12,26 @@ import { GRAPHQL_TARGETS } from '@commercetools-frontend/constants';
 import deprecateComponent from '../../from-core/deprecate-component';
 import ProjectQuery from './fetch-project.graphql';
 
+/**
+ * NOTE:
+ *
+ * Permissions are being fetched though the `allAppliedPermissions` which
+ * is an array of `{ name: string, value: boolean }`. This gives more
+ * flexibility to introduce new permissions to apps without having to release
+ * the merchant-center-app-kit.
+ *
+ * The application below however expects permissions to be of the shape
+ * `[name: string]: boolean` which is what the shape above is mapped into here.
+ */
+export const mapAppliedPermissionsToPermissions = appliedPermissions =>
```

Comment:
Would it make sense to move this function into the `permissions` package instead?

Reply:
True. Maybe it makes sense to expose it too.

Reply:
Sure, was just a suggestion 👌

Reply:
I value it! It's a really cool one. Maybe I'll add a note to make sure that we remind ourselves why this ain't moved yet.